gammapy/image/maps.py

```diff
@@ -252,6 +253,14 @@ def solid_angle(self):
         omega = -np.diff(xsky, axis=1)[1:, :] * np.diff(ysky, axis=0)[:, 1:]
         return Quantity(omega, 'deg2').to('sr')
 
+    def center(self):
+        """
+        Center coordinates of the sky map.
```

Comment:
Add info about what is returned:
```
Center coordinates of the sky map (`~astropy.coordinates.SkyCoord`).
```
I'm never sure what a good guideline is ... should we make methods without arguments properties?
Or leave them as methods in case we want to add keyword arguments without breaking callers later?